packages/react-vapor/src/components/limit/Limit.tsx

```diff
@@ -19,98 +20,69 @@ export interface LimitOwnProps {
     isLimitEditable?: boolean;
     isLimitTheGoalToReach?: boolean;
     isHistoryIncluded?: boolean;
+    onClick?: () => void;
```

Comment:
I would suggest renaming that prop to `onHistoryIconClick` to make it more obvious
```suggestion
    onHistoryIconClick?: () => void;
```